recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ package:
 
 source:
   - url: https://github.com/NVIDIA/nvmath-python/archive/refs/tags/v{{ ".".join(version.split(".")[:3]) }}.tar.gz
-    sha256: 155b60e6dbacfea617aa9a2e92d262062916ccc2e3b51af199d4c1e2d9ef6555
+    sha256: aff642b16b7dea06d34093bf4dc1b2f4f88e75382e1188423640c1cdb8d2bc0b
     patches:
       - 000-curand-path.patch
       - 001-filter-source-files.patch
```

Comment:
Looks like these no longer apply on the new release. Hence the CI failures

Do they need to be reworked or can they be dropped?